test/configuration.test.js

```diff
@@ -28,6 +28,11 @@ const options = [{
   default: 4194304,
   test: 5,
   alt: 555
+}, {
+  name: 'smoothQualityChange',
+  default: undefined,
```

Comment:
should the default be `false` instead of `undefined`?

Reply:
Good question. [Technically speaking, I don't set the default value to `false`, I just let it be undefined if it's not defined in the options.](https://github.com/videojs/http-streaming/pull/235/files#diff-24fe5b9b63a0e1c36b46a09177d8027fR388) So I chose `undefined` here since that's what the default would be in absence of an explicit value. Do you think I should explicitly set the option to be `false` if a value isn't given, or just change the default here to be `false`?

Reply:
I think it would be good to set it to false if not set like we do with `withCredentials` [here](https://github.com/videojs/http-streaming/blob/master/src/videojs-http-streaming.js#L337) The explicit setting of the false default should be done here as well.

Reply:
Yup, fair enough, explicit is always good.